source/ed/docker_tests/ed_integration_tests.cpp

```diff
@@ -426,3 +426,41 @@ BOOST_FIXTURE_TEST_CASE(poi2ed_pois_uri_should_match_mimir_naming, ArgsFixture)
             pois, [&](const navitia::georef::POI* p) {return p->uri == "poi:PAIHABIT0000000028850973";}
     ));
 }
+
+BOOST_FIXTURE_TEST_CASE(ntfs_dst_test, ArgsFixture) {
+    const auto input_file = input_file_paths.at("ntfs_dst_file");
+    nt::Data data;
+
+    bool failed = false;
+    try {
+        data.load_nav(input_file);
+    } catch(const navitia::data::data_loading_error&) {
+        failed = true;
+    }
+    data.build_raptor();
+    BOOST_REQUIRE_EQUAL(failed, false);
+
+    BOOST_REQUIRE_EQUAL(data.pt_data->lines.size(), 1);
```

Comment:
Why do you require `lines` and `routes` to have a size of 1 when you don't access them ? Maybe they are `BOOST_CHECK_*` instead ?